.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report 🐛
+about: Report errors or unexpected behavior
+title: "Bug Report"
+labels: Issue-Question
```

Comment:
I left this as `Issue-Question` because we need to triage it as an accepted bug. Is that reasonable?

Reply:
I'm sort of inclined to leave it with no label rather than auto-bucketing to question in that case.

Reply:
```suggestion
labels: 
```